### DISCUSSION TITLE
Der download der OpenAndroMaps Karten elend langsam 😭

### DISCUSSION THREAD

**Discussion by SF4User** [devemux86/cruiser#648] [Q&A] [answered]
Hallo! 
Vor Kurzem war schon ein Update der Offline-Karten nötig, nun schon wieder (OpenAndroMaps & MapsForge).
Eigentlich ja kein Problem. Nun dauert aber das Upload der OpenAndroMaps schon wieder, wie schon vorher, Tage ! um acht Europa-Karten und die Alps herunter zu laden.
Ich lasse das MobilPhone für den Down Load  immer die ganze Nacht durchlaufen . . . 2-3 Karten pro Nacht dauert das . . . also im Schnitt 3 Tage !!!
Download im 5.0 im GHz WLAN (ADSL Geschwindigkeit: ↓ 61,7 Mbit/s ↑ 26,4 Mbit/s) - der Download läuft aber nur mit etwa 256kBB/s.
Das bedeutet, daß der Download vom Server so elend langsam ist.
Nicht erst seit Neuem . . . eigentlich schon immer (also beim Cruiser). Ich hatte ja vorher den Kurviger #2 (die Cruiser Kombination) - da war das ned so extrem. Wenn man mal unterwegs - ohne WLAN - eine Karte benötigt . . . undenkbar!
Was ist die Ursache? gibt es eine Abhilfe?
Danke für jede Information.

**Comment by r11d11** [comment]
Ich hatte in letzter Zeit keine Geschwindigkeitsprobleme mehr mit dem Kartendownload. Habe heute Mapsforge Österreich und Italien runtergeladen. Cruiser zeigt mir mir eine Downloadgeschwindigleit zwischen  3 und 10 MByte/s an. Das ist für mich sehr gut, weil viel mehr geht über meine 100Mbit Leitung nicht.  Ich weiß, dass  hilft dir bei deinem Problem nicht weiter aber um das Problem einzukreisen ist ,wie du schreibst, jede Information hilfreich. Cruiser 5.0.1.b10 mit Android 11 und 5Ghz Wlan

**Comment by devemux86** [comment]
Downloads are done via the Android download manager, you can check them in Android status bar.
The app has nothing to do with downloads, the Android system handles the downloads and speed.
Maybe this is a local problem, as all other users can download with max speed.
Note that downloading a map, it also downloads its poi database and optionally BRouter and hillshading data.
(which are usually very small)

Downloads werden über den Android-Download-Manager durchgeführt, Sie können sie in der Android-Statusleiste überprüfen.
Die App hat nichts mit Downloads zu tun, das Android-System kümmert sich um die Downloads und die Geschwindigkeit.
Vielleicht ist dies ein lokales Problem, da alle anderen Benutzer mit maximaler Geschwindigkeit herunterladen können.
Beachten Sie, dass das Herunterladen einer Karte, es lädt auch seine poi Datenbank und optional BRouter und hillshading Daten.
(die aber in der Regel sehr klein sind)

**Reply by SF4User** [reply]
Ich habe mein Samsung A72 von hinten nach vorne abgesucht . . . Auf einen Android-Download-Manager habe ich nirgens Zugriff. So etwas gibt es auf meinem Samsung nicht - jedenfalls nicht über einen User Zugriff. In System mag das ja sein, aber wie bekomme ich den Android-Download-Manager dazu,  dass er besser performed? Beim App Update über den Google Play Störe geht es eigentlich immer recht flott.

**Reply by devemux86** [reply]
Android download manager is a service that is available in all devices and is provided by the Android system.
Apps use it to download files. Apps and users cannot influence it, it simply works and is managed by Android.

Der Android-Downloadmanager ist ein Dienst, der in allen Handys vorhanden ist und vom Android-System bereitgestellt wird.
Apps nutzen ihn, um Dateien herunterzuladen. Apps oder Benutzer können ihn nicht beeinflussen, er funktioniert einfach und wird von Android verwaltet.

**Reply by SF4User** [reply]
Also, ich glaube da nicht an die Story mit dem Android-Downloadmanager. Tägliche Versuche mit dem Download sind sehr unterschiedlich. Gestern konnte ich die Alps-Karte relativ schnell herunterladen, mit 4,5 . . . 6,2MByte/s (das wäre schon einigermaßen OK). Heute der nächste Versuch: Austria & Schweiz Karte: zwischen 0kByte/s . . . 64kByte/s - schon den ganzen Vormittag.
Es geht also heute (Sonntag) nichts. Daß mein Android-Downloadmanager täglich sein Verhalten ändert, das glaube ich absolut nicht - zumal der Download von anderen Dingen (App's vom Google Play z.B.) ganz normal funktioniert. Ich denke der Cruiser Server hat da ein Problem. Jetzt, 12:45h Download: 0kB/s . . . 0%. Egal ob WLAN oder mobiles LTE

**Reply by devemux86** [reply]
Offline maps are provided by their own servers.

Offline-Karten werden von ihren eigenen Servern bereitgestellt.

**Reply by devemux86** [reply]
I tried the downloads again:
- **Mapsforge**: maximum speed
- **OpenAndroMaps**: currently slow, but there's nothing we can do, just try some other time

---

Ich habe die Downloads erneut versucht:
- **Mapsforge**: maximale Geschwindigkeit
- **OpenAndroMaps**: derzeit langsam, aber wir können nichts tun, versuchen Sie es einfach ein anderes Mal

**Reply by SF4User** [reply]
Mapsforge Karten sind alle up to date . . . da hat der Update im Hintergrund offenbar easy funktioniert 👍
Das Problem ist offenbar nur bei den OpenAndroMaps (Server). Momentan benutze ich die Navigation ja nicht - Saison Ende . . . also nicht wirklich ein "Problem".
Ich halte meinen Cruiser nur immer schön up to date . . . weil mir ist es schon mal passiert (war es Calimoto? oder Kurviger 2?),
daß ich im Irgendwo keine aktuellen Karten hatte, und dann auch keine funktionierende Navigation. Frage: ist das immer noch so?
Wenn ich keine aktuellen Karten habe, funktioniert die Navigation normal? dann halt mit den "alten" Karten? Wenn das so ist, dann erspare ich mir das regelmäßige down load . . . oder eben nur 1x . . . 2x im Jahr am Anfang der Saison. Danke übrigens, für die Antworten 👍. Ich mag den Cruiser! Calimoto & Kurviger benutze ich zwar auch, aber nur wegen der Anderen Buddie's im Club.

**Reply by devemux86** [reply]
Offline maps usually only need to be downloaded once. You do not need to update them very often.
Navigation always works. If you use BRouter (offline), you can update it often, it has small downloads.

Offline-Karten müssen in der Regel nur 1 Mal heruntergeladen werden. Sie müssen sie nicht sehr oft aktualisieren.
Die Navigation funktioniert immer. Wenn Sie BRouter (offline) verwenden, können Sie es oft aktualisieren, es hat kleine Downloads.

**Comment by devemux86** [comment]
For the speed problems with the OpenAndroMaps server, it is better to report them in the [OpenAndroMaps forum](https://www.openandromaps.org/en/oam-forums).

Für die Geschwindigkeitsprobleme mit dem OpenAndroMaps Server, ist es besser, sie im [OpenAndroMaps Forum](https://www.openandromaps.org/oam-forums) zu melden.

**Comment by devemux86** [comment]
https://www.openandromaps.org/en/we-habe-been-hacked-again

https://www.openandromaps.org/wir-wurden-wieder-mal-gehacked

**Reply by kds112** [reply]
Yes, what a pitty... But the download is performed by the external ftp server that is hosted separately from the attacked website.

**Comment by devemux86** [comment]
https://www.openandromaps.org/en/download-server-lagging

https://www.openandromaps.org/schwierigkeiten-mit-gwdg-download-server

**Comment by devemux86** [comment, marked as answer]
OpenAndroMaps speed seems to be fixed.

Die Geschwindigkeit von OpenAndroMaps scheint behoben zu sein.

**Reply by SF4User** [reply]
Ja! Das Download der aktuellen Karten hat recht gut funktioniert.
